lib/EbookTag.php

```diff
@@ -17,4 +22,61 @@ protected function GetUrl(): string{
 
 		return $this->_Url;
 	}
+
+	// *******
+	// METHODS
+	// *******
+
+	/**
+	 * @throws Exceptions\ValidationException
+	 */
+	public function Validate(): void{
```

Comment:
Should validate that name exists and is not null or empty. Name should also be trimmed here.

Reply:
Done, with the same comment about 

```php
$this->Name = trim($this->Name ?? '');
```

above.